qutip/qip/gates.py

```diff
@@ -488,6 +509,29 @@ def sqrtiswap(N=None, targets=[0, 1]):
                               [0, 0, 0, 1]]), dims=[[2, 2], [2, 2]])
 
 
+def MS_gate(theta, N=None, targets=(0, 1)):
+    """
+    Quantum obejct of a Mølmer–Sørensen gate
+
+    Parameters
+    ----------
+    theta : The duration of the interaction pulse
+    """
```

Comment:
Also here: Shouldn't it have also "Returns" in the docstring?

Reply:
Add ` target ` docstring. Maybe default should be a list instead of a tuple for coherence with rest of code where `integer` is integer or list?